.travis.yml

```diff
@@ -3,10 +3,15 @@ language: php
 php:
   - '7.1'
   - '7.2'
+  - '7.3'
   - nightly
 
+matrix:
+  allow_failures:
+    - 'nightly'
```

Comment:
- Add the `php-7.3` version test because this version is stable.
- Let the `nightly` version allow to be failed because nobody can guarantee that the unstable version will always be successful during every Travis CI build.